usr/src/uts/armv8/boot/boot_early_uart.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file and its contents are supplied under the terms of the
+ * Common Development and Distribution License ("CDDL"), version 1.0.
+ * You may only use this file in accordance with the terms of version
+ * 1.0 of the CDDL.
+ *
+ * A full copy of the text of the CDDL should have accompanied this
+ * source.  A copy of the CDDL is also available via the Internet at
+ * http://www.illumos.org/license/CDDL.
+ */
+
+/*
+ * Copyright 2025 Michael van der Westhuizen
+ */
+
+#ifndef _BOOT_EARLY_UART_H
+#define	_BOOT_EARLY_UART_H
+
+/*
+ * Debug-only fixed UART definitions for platform bringup and troubleshooting.
+ */
+
+#if !defined(_EARLY_DBG_UART)
+#define	_EARLY_DBG_UART	0
+#endif
+
+#if defined(_EARLY_DBG_UART) && _EARLY_DBG_UART > 0
+/*
+ * qemu sbsa-ref is: 0x60000000ULL, 0x000eULL
+ *     qemu virt is: 0x09000000ULL, 0x000eULL
```

Comment:
I don't remember off the top of my head where raspberry pi is, but we should probably list it

Reply:
I think this will be `0xfe201000`, but I'll double-check against the FDT.